what4/src/What4/Protocol/VerilogWriter/AST.hs

```diff
@@ -0,0 +1,375 @@
+{-
```

Comment:
This module could really use a lot more comments.

Reply:
That should be doable!